### PR TITLE
add advice on running meetings

### PR DESCRIPTION
**Pull request by SethTisue**
this is mostly drawn from an email Jon sent me about how to
chair meetings, and from an email I sent Rob about taking
notes

### CODE REVIEW

**Comment by SethTisue** [comment]
I think @stuhood will want to see this. GitHub won't let me formally review-request him; @sjrd could you add him as a collaborator on this repo? (I don't have the necessary rights.)

@propensive I drew heavily on the text you sent me about chairing the meeting, since that was the pressing matter at the time. But before much longer, the next pressing thing will be for @stuhood to organize the next meeting. Perhaps you could add a section to this with advice on that...? (And anything else from your experience as chair that might be helpful to others, down the road?)

**Comment by sjrd** [comment]
> @sjrd could you add him as a collaborator on this repo? (I don't have the necessary rights.)

Done, but it's pending acceptation of the invite.

File: meetings.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+acting chair) and the secretary.
+
+## Chairing
+
+Some miscellaneous advice, in no particular order, and much of it is
+perhaps obvious anyway:
+
+### Before
+
+* Have an idea for how long each part of the meeting should take.
```
Comment (stuhood) [inline meetings.md]: Would suggest having an agenda document for the meeting, bootstrapped with any of the required business.

Comment (SethTisue) [reply meetings.md]: commit added that addresses this.

**Comment by SethTisue** [comment]
I've emailed Jon to see if he has anything to add before we merge this.

**Comment by propensive** [comment]
Regarding the task of organizing the meeting, for the benefit of the chairperson, I would suggest that the scheduling of the meeting between attendees is something that the Scala Center could do, instead of the chairperson,  possibly with an invitation to the chair to include any notes/advice/recommendations to include in the emails.

Although I never dropped the ball on organizing any of the meetings, I occasionally left it later than I would have liked, and I think it might have have been less convenient for those members who needed to make plans. Not to suggest in any way that @stuhood or another chair would be as disorganized as me, but having the burden of scheduling the meeting was something to which I'm not naturally suited, and having this part of the job handed over to the Scala Center would allow the chairperson to focus more on the *chairing* part of the role.

File: meetings.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+### Before
+
+* Send out the meeting agenda a few days ahead of time, to give
+  the board time to consider topics in advance.
+* The chair or secretary should also upload the agenda to
+  https://github.com/scalacenter/advisoryboard/tree/master/agendas
+* Before the meeting begins, review the agenda and arrive at a rough
+  idea of how long each part of the meeting should take, in order to
+  fit everything in.
+
```
Comment (propensive) [inline meetings.md]: Further advice I would add:
```
The person responsible for scheduling the meeting should try to find a time suitable for everyone, though it's rarely possible that everyone can attend any given meeting. Priority should be given to the SC executive director, the technical adviser, the chairperson and the secretary, without whom the meeting can't really happen. If some other attendees are unable to attend on a particular time, priority should be given to those who are presenting proposals, and at the very least, an individual email should be sent to those members who are unavailable to check if they can delegate to someone else. With all else equal, any remaining toss-up between two dates should probably be decided in favor of any members who missed the previous meeting, and then in favor of those members who offered the greatest flexibility on their availability.

The role of being an Advisory Board representative is not always at the top of anyone's agenda, and consequently, members can forget to make proposals to the board. Without proposals to discuss, the meetings lose some of their relevance, so a part of chairperson's role should be to solicit and encourage proposals for discussion.
```

File: meetings.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+### Starting
+
+* Before beginning the meeting, start the recording.
+* Make sure the recording clearly indicates who was present and
+  which board members were absent, if any.
```
Comment (propensive) [inline meetings.md]:
```
Permission to make the recording should be sought from everyone present.
```

**Comment by SethTisue** [comment]
thx Jon, I incorporated the material you suggested.

**Pull request merged**